tools/depends/configure.ac

```diff
@@ -194,13 +194,14 @@ case $host in
     use_sdk="${use_sdk:-android-24}"
     use_ndk_api="${use_ndk_api:-21}"
     deps_dir="$use_host-$use_ndk_api-$build_type"
-    platform_cflags="-DANDROID -D__ANDROID_API__=$use_ndk_api -fexceptions -fPIC -DPIC -D_GLIBCXX_USE_C99_MATH_TR1"
+    platform_cflags="-DANDROID -D__ANDROID_API__=$use_ndk_api -fexceptions -funwind-tables -fstack-protector-strong -no-canonical-prefixes  -fPIC -DPIC -D_GLIBCXX_USE_C99_MATH_TR1"
     optimize_flags="-Os"
     if test "x$use_cpu" = "xarm64-v8a"; then
       platform_cflags+=" -march=armv8-a -mtune=cortex-a53"
     fi
+    platform_ldflags="-Wl,--exclude-libs,libgcc.a -Wl,--exclude-libs,libatomic.a -L$prefix/$deps_dir/lib/android-$use_ndk_api"
     platform_ldflags="-L$prefix/$deps_dir/lib/android-$use_ndk_api"
```

Comment:
remove this line? Or maybe dont touch arm !=32 ? Not sure if NDK adds the -Wl things only to 32 bit

Reply:
Typo. 
NDK add specifically the libunwind for arm32 only, but the other 2 for all